e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_1_spec.js

```diff
@@ -441,6 +441,16 @@ describe('Keyboard Shortcuts', () => {
         cy.get('#moreDmModal').should('not.exist');
     });
 
+    it('MM-T4452 - CTRL/CMD+SHIFT+. Expand or collapse RHS', () => {
+        // # Type CTRL/CMD+SHIFT+. to open 'RHS'
+        cy.get('#post_textbox').cmdOrCtrlShortcut('{shift}.');
+        cy.uiGetRHS();
+        cy.get('#post_textbox').cmdOrCtrlShortcut('{shift}.');
```

Comment:
Although not documented in our dev docs, for sake of clarity we comment what each command does

https://github.com/mattermost/mattermost-webapp/blob/52c99a026a3c57a8c6e7ac3ff001a122ec850200/e2e/cypress/integration/keyboard_shortcuts/keyboard_shortcuts_1_spec.js#L5-L6

It makes it easier to read the comments and figure out the test steps without understanding the code